frontend/src/routes/docs/rooc/+page.svelte

```diff
@@ -56,6 +69,24 @@
             <a href="/docs/rooc/examples">Rooc examples</a>
         </li>
     </ul>
+
+    <h1>Why Would You Use ROOC?</h1>
+    <p>
+    Picture your team dealing with limited resources and an ever-growing workload. It's not long before you're searching for an SaaS like Linear app to help divide up the work and track overlap between responsibilities and problems.
```

Comment:
I like the idea, I think we should try to find a "common" problem (which is easy to model) that we can then formalize into a known model (for example how was done for the diet, but that's a bit too complicated for an introduction" and then show step by step the mentality of creating a model, without going too in depth 

Then we can have another page where it explains how optimization works in general, with more examples, and then another page with a "how to model a model", but let's do that outside this PR

Reply:
What do you think about something like this:

---

You have a $50 store coupon and need to buy groceries for the week. Your shopping list includes:

- At least 3 fruits
- At least 2 vegetables
- At least 1 dairy product
Each item has a cost:

- Apples: $4 each
- Bananas: $3 each
- Carrots: $2 each
- Spinach: $3 per bunch
- Milk: $5 per carton
- Cheese: $6 per block

Your goal is to select items that meet the requirements and spend at least the full coupon, but you want to minimize spending more than the coupon value.

Reply:
I think that works as introductory example for now. It could still be very powerful to include an example where your contraints include human resources, like a small startup team. That's something I feel a lot of people can relate to with MILP problems. 

But we could decide later if we want to expand on the intro page or leave other examples for elsewhere deeper into the docs. For now the grocery store example works as a way to tie ROOC to a real-life scenario.

Reply:
Yes we can add an example for that in another page